services/svc-pinger/test/ping.controller.test.ts

```diff
@@ -0,0 +1,107 @@
+import axios from "axios";
+import { pingController } from "../src/controllers/ping.controller";
+import { describe, it, expect, vi, beforeEach } from "vitest";
+
+vi.mock("axios")
+
+const mockRes = () => {
+    const res: any = {}
+    res.status = vi.fn().mockReturnValue(res)
+    res.json = vi.fn().mockReturnValue(res)
+    res.sendStatus = vi.fn().mockReturnValue(res)
+    return res
+}
+
+describe("pingController", () => {
```

Comment:
To avoid unnecessary duplication of tests, the controller should be tested within the integration tests of corresponding endpoints